cmds/tui.c

```diff
@@ -314,9 +431,9 @@ static struct display_field graph_field_addr = {
 
 /* index of this table should be matched to display_field_id */
 static struct display_field *graph_field_table[] = {
-	&graph_field_total,
-	&graph_field_self,
-	&graph_field_addr,
+	&graph_field_total,	&graph_field_total_avg, &graph_field_total_min,
+	&graph_field_total_max, &graph_field_self,	&graph_field_self_avg,
+	&graph_field_self_min,	&graph_field_self_max,	&graph_field_addr,
```

Comment:
I think it's better to split this part into another commit and to focus on self time calculation here.

Reply:
Hmm.. you didn't do that.

Anyway, please consider updating the documentation too.

Reply:
@namhyung 
I was told to add a function corresponding to each commit.
 Wasn't it added to the two commits?

Reply:
I expect we would have 4 commits at the end.

Reply:
Please see my earlier comment.

> I think your work contains the following changes.
> 
> 1.  convert node->total_time to struct graph_time_stat
> 2.  change node->child_time to node->self_time (and make it struct graph_time_stat)
> 3.  add avg/max/min to uftrace graph command
> 4.  add avg/max/min to uftract tui command
>
> Each change can be in a different commit.

Reply:
@namhyung 

Oh, I'm sorry that I missed it.